tests/unit/falcon_development/DevelopmentTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace falcon_development;
+
+use Drupal\user\Entity\Role;
+
+class DevelopmentTest extends \Codeception\Test\Unit
+{
+    /**
+     * @var \UnitTester
+     */
+    protected $tester;
+
+    /**
+     * Checks that specified dev modules are enabled.
+     */
+    public function testDevelopmentModulesEnabled()
+    {
+        foreach ($this->tester->getDevelopmentModules() as $module) {
+            $this->assertTrue(
+                \Drupal::moduleHandler()->moduleExists($module),
+                "Module $module should be enabled."
+            );
+        }
+    }
+
+    /**
+     * Checks that test users esist and enabled.
+     */
+    public function testTestUsersEnabled()
```

Comment:
I'm soooo excited !

Reply:
@spleshka we can use the same Drupal & DB modules with all kinds of tests! :)